README.md

```diff
@@ -1,285 +1,125 @@
-<a name="readme-top"></a>
+# BUDGET APP
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
+Budget- App is a Ruby on Rails mobile web application that allows users to keep track of their expenses and categories. have a list of transactions associated with a category, so that the user can see how much money it spent and on what. Manage your own money, manage your own life.
 
-Modify this file to match your project and remove sections that don't apply.
+## Built With
 
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
+- Ruby
+- Rails
+- Postgresql
+- Linters
 
-After you're finished please remove all the comments and instructions!
--->
+### Key Features
 
-<div align="center">
+- [Log in and registration of the user]
+- [Recipe list ,Food list, Public recipe list pages]
+- [Functionality to Generate a shopping list with total cost]
 
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
+## Live Demo
+- To be deployed
```

Comment:
Please deploy the app online as it's requested by previous code reviewer. You can use any other service to host it and one of them is Netlify.

Reply:
Thank you for this, but Netlify is not deploying ROR projects
<img width="945" alt="image" src="https://user-images.githubusercontent.com/106079814/204058702-1c525ae0-4b6e-4498-842f-bd1efb877f6a.png">